src/main/java/bookstore/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package bookstore.controller;
+
+import bookstore.dto.user.UserRegistrationRequestDto;
+import bookstore.dto.user.UserResponseDto;
+import bookstore.exception.RegistrationException;
+import bookstore.service.user.UserService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/auth")
```

Comment:
add server.servlet.context-path=/api to application.properties
```suggestion
@RequestMapping("/auth")
```